changelog/unreleased/issue-2498

```diff
@@ -0,0 +1,5 @@
+Change: Fix inconsistent acces to config file
```

Comment:
Typo: acces. Also inconsitent, below.

Reply:
I removed the changelog for this minor change.